src/extension.ts

```diff
@@ -418,6 +432,17 @@ function launchMetals(
         case "metals-model-refresh":
           compilationDoneEmitter.fire();
           break;
+        case "metals-open-window":
+          const openWindowParams =
+            params.arguments && (params.arguments[0] as MetalsOpenWindowParams);
```

Comment:
```suggestion
          const openWindowParams = params.arguments?[0] as MetalsOpenWindowParams;
```

Reply:
Changed to `params.arguments?.[0] as MetalsOpenWindowParams;`